lib/graphql/schema.ts

```diff
@@ -1014,9 +1015,7 @@ export enum TransactionProjectionType {
   Rebooking = 'REBOOKING',
   CancellationDirectDebit = 'CANCELLATION_DIRECT_DEBIT',
   CancellationSepaCreditTransferReturn = 'CANCELLATION_SEPA_CREDIT_TRANSFER_RETURN',
-  CardTransaction = 'CARD_TRANSACTION',
-  InterestAccrued = 'INTEREST_ACCRUED',
-  CancellationInterestAccrued = 'CANCELLATION_INTEREST_ACCRUED'
```

Comment:
Maybe you didn't mean to remove this stuff

Reply:
This was added from us recently, we just merged a huge feature branch to develop along with a few other schema changes (and we didn't update the sdk schema for it yet).

I'm fine with both:
* only including "splits" change in this PR
* doing a bulk schema update with everything missing. In which case doing `schema:generate` against a fresh develop would reveal that these two lines should not be removed, as well as adding a few other overdraft related things to the schema

Reply:
ok, I will update to latest develop and regenerate the schema

Reply:
unless you want to do it earlier for any reason? @hihuz I mean this PR might be hanging here for a while since @shotexa might need to extend it working on his tickets in the scope of the same `split` feature. We just wanted to release everything split-related to SDK all at once under a single version

Reply:
We don't really need to do it any time soon, as overdraft will be available only from the native app, which doesn't use typescript for now 🙁so the schema only slightly helps for autocompletion there.

But on the other hand it's our responsiblity to do it, I'll create a PR to add our overdraft Schema changes 👍

Reply:
Added overdraft changes in this PR: https://github.com/kontist/js-sdk/pull/87 🙂